test/run.sh

```diff
@@ -71,20 +76,28 @@ if [ -z "${TESTS}" ]; then
     TESTS="./suites/standard"
 fi
 
+# if stress testing is set
```

Comment:
So when a stress test scenario is active, a separate RF file is run before the regular tests to enable it? Is that always going to make it possible to configure the host? I would think we would need to create the VM differently for different RAM settings, for example.

Reply:
Hi, this Stress Testing approach is independent of the requirements of the VM. You can enable/disable the Stress condition at any given moment and at any given time before running any test suite.

The separate RF file will enable the Stress condition before running any set of tests. Then, when tests are finished, the Stress condition will be disabled.

My idea is that will be always possible to configure the host, no need to change the way the VM is created at all. Any Stress condition can be enabled and disabled at any time.